includes/abstracts/abstract-wc-stripe-payment-gateway.php

```diff
@@ -21,7 +21,21 @@ abstract class WC_Stripe_Payment_Gateway extends WC_Payment_Gateway_CC {
 	 */
 	public function display_admin_settings_webhook_description() {
 		/* translators: 1) webhook url */
-		return sprintf( __( 'You must add the following webhook endpoint <strong style="background-color:#ddd;">&nbsp;%s&nbsp;</strong> to your <a href="https://dashboard.stripe.com/account/webhooks" target="_blank">Stripe account settings</a>. This will enable you to receive notifications on the charge statuses.', 'woocommerce-gateway-stripe' ), WC_Stripe_Helper::get_webhook_url() );
+		$must_add = sprintf( __( 'You must add the following webhook endpoint <strong style="background-color:#ddd;">&nbsp;%s&nbsp;</strong> to your <a href="https://dashboard.stripe.com/account/webhooks" target="_blank">Stripe account settings</a>. This will enable you to receive notifications on the charge statuses.', 'woocommerce-gateway-stripe' ), WC_Stripe_Helper::get_webhook_url() );
+
+		$last_event_time = get_option( 'wc_stripe_last_event_time', null );
+		if ( $last_event_time ) {
+			$webhook_status = sprintf(
+				/* translators: 1) time of last event received */
+				__(
+					'Event time of last webhook received: %s',
```

Comment:
Could we go shorter here? e.g. "Last webhook received at XXX"

Reply:
I'd opted to record and display the time _on the event_ – so it can be matched directly with a time in the Stripe dashboard. This would make a difference especially when a event is resent. I was attempting to be brief while also technically correct in the description (though I suppose it can still be shortened to "Time of last webhook received" or something).

Do you think the time displayed should rather be the actual server time when the last webhook is received?

Reply:
Oh, I wasn't recommending changing the %s at all - just the text leading up to it.

Reply:
> Do you think the time displayed should rather be the actual server time when the last webhook is received?

You're thinking time zone might be a factor here? If we can present the time of last webhook in the WordPress time zone that would be awesome. If not, if we can show the time zone from stripe (e.g. UTC) that will do.

Reply:
My concern was that if we don't change what the `%s` is (event timestamp), then it could be misleading to say that it was received at that time – mildly in the nominal case, severely for resent events.

I don't _think_ timezone is a factor since the event time should be UTC and the `date` function should localize it.